packages/create-plugin/src/commands/update.migrate.command.ts

```diff
@@ -0,0 +1,31 @@
+import minimist from 'minimist';
+import { gte } from 'semver';
+import { getMigrationsToRun, runMigrations } from '../migrations/manager.js';
+import { getConfig } from '../utils/utils.config.js';
+import { getVersion } from '../utils/utils.version.js';
+import { printHeader } from '../utils/utils.console.js';
+
+export const migrationUpdate = async (argv: minimist.ParsedArgs) => {
+  try {
+    const projectCpVersion = getConfig().version;
+    const packageCpVersion = getVersion();
+
+    if (gte(projectCpVersion, packageCpVersion)) {
+      console.warn('Nothing to update, exiting.');
+      process.exit(0);
+    }
+
+    console.log(`Running migrations from ${projectCpVersion} to ${packageCpVersion}.`);
+
+    const commitEachMigration = argv.commit;
+    const migrations = getMigrationsToRun(projectCpVersion, packageCpVersion);
+    await runMigrations(migrations, { commitEachMigration });
+    printHeader('the update command completed successfully.');
+    console.log('');
+  } catch (error) {
+    printHeader('the update command encountered an error.');
```

Comment:
add a note here to instruct the user they can run `git reset hard --origin [ref]` (calculate the ref using git) so users don't get lost in the case of error.

Maybe even add a link to our github repository for them to report the issue?